src/pages/indexer/IndexerProfile/IndexerProfile.tsx

```diff
@@ -189,7 +189,7 @@ const ActiveCard = (props: { account: string }) => {
 const IndexerProfile: FC = () => {
   const { id: account } = useParams();
   const checksumAddress = useMemo(() => {
-    return toChecksumAddress(account || '');
+    return toChecksumAddress(account || '0x0000000000000000000000000000000000000000');
```

Comment:
Update this as well, also be good to check the whole code base

Reply:
@mzxyz updated~.